assets/webconfig/i18n/en.json

```diff
@@ -365,7 +363,7 @@
 		"edt_append_hz" : "Hz",
 		"edt_append_pixel" : "Pixel",
 		"edt_append_percent" : "%",
-		"edt_append_degree" : "�",
+		"edt_append_degree" : "�",
```

Comment:
@penfold42 I think we should face the encoding problems instead of that ampersand "old way" :)